caom2-tap/build.gradle

```diff
@@ -16,7 +16,7 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '1.7.9'
+version = '1.7.11'
```

Comment:
version should be 1.7.10

dependency below should be 1.2.5

Reply:
Updated.